tests/framework/common.go

```diff
@@ -215,6 +215,8 @@ type OsmTestData struct {
 	Client          *kubernetes.Clientset
 	SmiClients      *smiClients
 	ClusterProvider *cluster.Provider // provider, used when kindCluster is used
+
+	DeployOnOpenShift bool // Determines whether to configure tests for OpenShift
```

Comment:
This could be a bit more generic and talk about SCC enabled cluster, vs OpenShift.

Reply:
Let's take more time to think about this and consider in a follow up PR